### PR TITLE
Code style updates

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

**Comment by hjmjohnson** [comment]
@thewtex  This should be ready for merging now.

We have MUCH more success! We are all the way to building and testing with 2/3 of the test passing.
```
67% tests passed, 271 tests failed out of 811

Label Time Summary:
RUNS_LONG    =  22.37 sec*proc (3 tests)
```

Mathew Seng will be working on fixing the tests next week if this PR is merged before next monday.

File: src/Core/Common/BresenhamLine/Code.cxx [hunk 1]
```diff
@@ -44,9 +44,9 @@ void Vector()
   v[1] = 1;
   std::vector< itk::Offset<2> > offsets = line.BuildLine(v, 4);
 
-  for(unsigned int i = 0; i < offsets.size(); i++)
+  for(auto offset : offsets)
```
Comment (dzenanz) [inline src/Core/Common/BresenhamLine/Code.cxx]: Commit message has an unfinished sentence: `C++11 Range based for loops can be used in`.

Comment (hjmjohnson) [reply src/Core/Common/BresenhamLine/Code.cxx]: Done. Thanks.

**Comment by thewtex** [comment]
> We have MUCH more success! We are all the way to building and testing with 2/3 of the test passing.
> 
> ```
> 67% tests passed, 271 tests failed out of 811
> 
> Label Time Summary:
> RUNS_LONG    =  22.37 sec*proc (3 tests)
> ```
> 
> Mathew Seng will be working on fixing the tests next week if this PR is merged before next monday.

Awesome! Great work Hans and Mathew!!

**Pull request merged**